### PR TITLE
Feature/loguru

### PR DESCRIPTION
**Pull request by shatteredbeam**
Removed all instances of LOG and custom logging to switch over to using loguru.

### CODE REVIEW

File: src/config/__init__.py [hunk 1]
```diff
@@ -78,4 +83,20 @@
...
+        # Intercepts standard logging messages for the purpose of sending them to loguru
+        depth = next(i for i, f in enumerate(inspect.stack()[1:]) if
+                     f.filename != logging.__file__) + 1
+        logger_opt = logger.opt(depth=1, exception=record.exc_info)
+        logger_opt.log(logging.getLevelName(record.levelno), record.getMessage())
+
+
+# Hook logging intercept
+logging.basicConfig(handlers=[InterceptHandler()], level=0)
+logger.info("Logging Hook Executed.  As a workaround, log messages sent to logging will be "
```
Comment (theunkn0wn1) [inline src/config/__init__.py]: [question] do we really still need this intercept handler? I thought by migrating everything to logaru we wouldn't need to do this?

**Review by Eearslya — DISMISSED** [review]

**Review by theunkn0wn1 — DISMISSED** [review]
LGTM, will do one final review (by running the code) tomorrow at the latest 😇

File: src/config/__init__.py (hunk 1 above)
Comment (shatteredbeam) [reply src/config/__init__.py]: Yep.  It'll pull logging messages from other input sources (ahem, Pydle) and push them into our instance.

**Review by theunkn0wn1 — APPROVED** [review]
👍 LGTM

**Review by Eearslya — APPROVED** [review]

**Pull request merged**